application/binary/build.gradle.kts

```diff
@@ -23,5 +25,44 @@ androidBinary(
         project(":common:progressbar:databinding"),
         project(":core:mvvm:library"),
         project(":core:di:library")
+    ),
+
+    /**
+     * Only for :binary-debug
+     */
+    buildConfiguration = BuildConfiguration(
+        name = DebugBuildName(),
+        singinConfig = DebugeSigningConfig(
+            keystoreFile = KeystoreFile("debug-keystore.properties")
+        )
+    ),
+
+    /**
+     * Only for :binary-release
+     */
+    buildConfiguration = BuildConfiguration(
+        name = ReleaseBuildName(),
+        singinConfig = ReleaseSigningConfig(
+            keystoreFile = KeystoreFile("release-keystore.properties")
+        ),
+        optimizeConfig = OptimizeConfig(
+            minificationFiles = setOf("proguard-rules.pro"),
+            shrinkCode = true,
+            shrinkResorces = true
+        )
+    ),
+
+    /**
+     * For :binary-custom
+     */
+    buildConfiguration = BuildConfiguration(
+        name = BuildName(
+            name = "custom",
+            appIdSuffix = ".custom"
+        ),
+        singinConfig = SigningConfig(
+            name = "Custom",
```

Comment:
Why we heed a name here? I don't think we want to reuse is somwhere

Reply:
Yea, I checked. We don't need to pass additional name for configs, especially for different binaries.